src/users/users.service.ts

```diff
@@ -1,58 +1,48 @@
 import {
   BadRequestException,
+  ConflictException,
+  ForbiddenException,
   Injectable,
   InternalServerErrorException,
 } from "@nestjs/common";
-import { UserRepository } from "./users.repository";
+import { UsersRepository } from "./users.repository";
 import { GetUsersByAnimalDto } from "./dtos/get-users-by-animal.dto";
 import { PrismaService } from "src/prisma/prisma.service";
-import { InventoryRepository } from "src/inventory/inventory.repository";
+import { UpdateUserNicknameDto } from "./dtos/update-user-nickname.dto";
+import { UpdateUserDescriptionDto } from "./dtos/update-user-description.dto";
+import { DomainEnum } from "./enum/domain-enum";
 
 @Injectable()
 export class UsersService {
   constructor(
     private readonly prisma: PrismaService,
-    private readonly userRepository: UserRepository,
-    private readonly inventoryRepository: InventoryRepository,
+    private readonly userRepository: UsersRepository,
   ) {}
 
-  async getUserNameCurrentPointAccumulationPointTitle(userNo: number) {
-    const result =
-      await this.userRepository.getUserNameCurrentPointAccumulationPointTitle(
-        userNo,
-      );
-
-    return {
-      nickname: result.nickname,
-      currentPoint: result.currentPoint,
-      accumulationPoint: result.accumulationPoint,
-      title: result.userAchievement[0].achievement.title,
-      fontColor: result.userAchievement[0].achievement.level,
-    };
+  getUserNamePointTitleCharacter(userNo: number) {
+    return this.userRepository.getUserNamePointTitleCharacter(userNo);
   }
 
-  // async createUser(
-  //   uniqueIdentifier: string,
-  //   socialName: string,
-  //   image: string,
-  //   domain: string,
-  // ) {
-  //   const result = await this.userRepository.createUser(
-  //     uniqueIdentifier,
-  //     socialName,
-  //     image,
-  //     domain,
-  //   );
-  //   return result;
-  // }
+  async createUser(
+    uniqueIdentifier: string,
+    socialName: string,
+    image: string,
+    domain: DomainEnum,
+  ) {
+    const result = await this.userRepository.createUser(
+      uniqueIdentifier,
+      socialName,
+      image,
+      domain,
+    );
+    return result;
```

Comment:
```suggestion
    );
    
    return result;
```
그리고 await도 필요 없고 바로 return 하면 되네요. 쓸데 없는 변수 선언은 지양하는게 좋습다